query/README.md

```diff
@@ -0,0 +1,146 @@
+Keen Query Clients
+===================
+
+[![Build Status](https://travis-ci.org/keenlabs/KeenClient-Java.png?branch=master)](https://travis-ci.org/keenlabs/KeenClient-Java)
+
+The query capabilities within the Java Keen client enable you to send POST queries and receive the results of the queries in a JSON object. For query types, refer to API technical reference: https://keen.io/docs/api/reference/.
+
+## Usage
+### Building a Keen Query Client
+You can build a KeenQueryClient by just providing a KeenProject.
+```java
+KeenProject queryProject = new KeenProject("<project id>", "<write key>", "<read key>");
+KeenQueryClient queryClient = new TestKeenQueryClientBuilder(queryProject).build();
+```
+Optionally, users can also specify a HTTP Handler, base URL, or JSON Handler:
+```java
+KeenQueryClient queryClient = new TestKeenQueryClientBuilder(queryProject)
+		.withHttpHandler(httpHandler)
+		.withJsonHandler(jsonHandler)
+		.withBaseUrl(baseURL)
+		.build();
+```
+### Using the KeenQueryClient to send Queries
+The most simple way that users can use the KeenQueryClient to send queries is as follows. These methods take only the required query parameters as input, and the user receives a very specific Integer or Double response type. Please note that we strongly encourage users to pass in the Timeframe parameter, but it can be null.
+```java
+Long count = queryClient.count("<event_collection>", new RelativeTimeframe("this_year"));
+Long countUnique = queryClient.countUnique("<event_collection>", "<target_property>", new AbsoluteTimeframe("2012-08-13T19:00:00.000Z","2015-06-07T19:00:00.000Z"));
+Double minimum = queryClient.minimum("<event_collection>", "<target_property>", new RelativeTimeframe("this_year"));
```

Comment:
Nit: Again, 1 year is a long timeframe for most of our customers. For the sake of anchoring expectations towards the lower end, maybe use `this_week` or `this_month`.